app/models/infra_conversion_job.rb

```diff
@@ -442,13 +440,25 @@ def poll_transform_vm_complete
       raise migration_task.options[:virtv2v_message]
     when 'succeeded'
       update_migration_task_progress(:on_exit)
-      queue_signal(:poll_inventory_refresh_complete)
+      queue_signal(:inventory_refresh)
     end
   rescue StandardError => error
     update_migration_task_progress(:on_error)
     abort_conversion(error.message, 'error')
   end
 
+  def inventory_refresh
+    update_migration_task_progress(:on_entry)
+    if migration_task.options[:destination_vm_uuid].present?
+      EmsRefresh.refresh(migration_task.destination_ems, :vms, migration_task.options[:destination_vm_uuid])
```

Comment:
You need to put the refresh request on the queue, what you have here will run the refresh and save_inventory right here, not on the Refresh Worker.  Check out `EmsRefresh.queue_refresh`